src/shogun/regression/svr/LibLinearRegression.h

```diff
@@ -147,7 +147,7 @@ class CLibLinearRegression : public CLinearMachine
 protected:
 
 		/** train machine */
-		virtual bool train_machine(CFeatures* data = NULL);
+		bool train_machine(CFeatures* data = NULL) override;
```

Comment:
Mixing using override and not using it causes warnings iirc

Reply:
ah ok, but at some point would probably start adding override no? Doesn't it come at a cost when you just add more and more virtual class members?

Reply:
Yes at some point. I would just leave this for another or or the detox gsoc project